Adjusted contract and requirement for Superrare

Kevkillerke opened the pull request:
Current address is the token address from superrare. It's only used for transfers: https://etherscan.io/address/0xba5BDe662c17e2aDFF1075610382B9B691296350

This address is for the bids/buys: https://etherscan.io/address/0x6d7c44773c52d396f43c2d511b81aa168e9a7a42
The type is changed to "transaction_from_address_count" because it's the contracts that sends you the NFT.

Kevkillerke committed:
Adjusted contract and requirement for Superrare

Current address is the token address from superrare. It's only used for transfers: https://etherscan.io/address/0xba5BDe662c17e2aDFF1075610382B9B691296350

This address is for the bids/buys: https://etherscan.io/address/0x6d7c44773c52d396f43c2d511b81aa168e9a7a42
The type is changed to "transaction_from_address_count" because it's the contracts that sends you the NFT.

blankey1337 commented:
In this situation, I think the contract address will be the `to` field: 
![image](https://user-images.githubusercontent.com/42594751/187050408-062784b5-d036-47c6-aade-e549504d4dc2.png)

We might end up catching inteactions besides "purchasing an NFT", but there doesn't seem to be a reliable method for distinguishing these contract interactions based on the data we are getting back from etherscan alone. We may want to updating the wording to say "Interact with Superrare". 

I think you are right about updating the contract address.

Kevkillerke committed:
changed superrare to own token requirement

All Superrare NFT's are from the same token address: https://etherscan.io/address/0xb932a70a57673d89f4acffbe830e8ed7f75fb9e0

Kevkillerke commented:
Updated the commit

Kevkillerke committed:
changed to "transaction_to_address_count"
